### ISSUE TITLE
Mapping between the class ID and the class name in Icon645 Resnet model

### ISSUE THREAD

**Issue by lekhang4497** [lupantech/IconQA#2]
I would like to use your pre-trained Resnet model on the Icon645 dataset for classification.

Using your pre-trained Icon645 Resnet model, I was able to output the 377-dimension softmax vector for classification (for 377 classes).

I would like to know the mapping from the class ID to the class name in your Icon645 Resnet model.
For example:
1 --> apple
2 --> acorn
...

Is this mapping available?
Thank you very much.

**Comment by lupantech** [comment]
Hi Nguyen, 

Thanks for your interest in our IconQA paper! The class names correspond to the alphabetical ascending orders of the 377 icon folders in `icon645.zip`. I uploaded the mapping file to the repo just now. Here is the link: https://github.com/lupantech/IconQA/blob/main/data/icon645_classes.json.

Thank you for reaching out! If you have any further questions, please let me know.

Best regards,

Pan

**Comment by lekhang4497** [comment]
Thank you very much for your quick response.
